internal/command/apps/create.go

```diff
@@ -74,6 +71,10 @@ func RunCreate(ctx context.Context) (err error) {
 		apiClient     = client.FromContext(ctx).API()
 	)
 
+	if flag.GetBool(ctx, "nomad") {
+		return fmt.Errorf("nomad platform is no longer supported")
```

Comment:
"... is deprecated" or "creation of an app on Nomad platform is ...". We still support the platform itself.